src/main/java/uk/gov/hmcts/reform/civil/handler/callback/camunda/docmosis/GenerateCUIResponseSealedFormCallBackHandler.java

```diff
@@ -47,16 +47,16 @@ public List<CaseEvent> handledEvents() {
     }
 
     private CallbackResponse prepareSealedForm(CallbackParams callbackParams) {
-        CaseData caseData = callbackParams.getCaseData();
```

Comment:
Why make 3 calls instead of using a variable that only makes one call? Is there a problem?

Reply:
done